pom.xml

```diff
@@ -99,12 +99,11 @@
         <h2.version>2.2.224</h2.version>
         <hibernate-orm.plugin.version>6.2.13.Final</hibernate-orm.plugin.version>
         <hibernate.c3p0.version>6.2.13.Final</hibernate.c3p0.version>
-        <infinispan.version>15.0.5.Final</infinispan.version>
+        <infinispan.version>15.0.6.Final</infinispan.version>
```

Comment:
@ahus1 Could you please confirm it is safe to upgrade ISPN from the SRE team perspective?

Reply:
@vmuzikar - we've chosen not to upgrade for now as the release contains an upgrade of JGroups which has a breaking change in the way it provides metrics. So I'd say we shouldn't upgrade to this one and wait for another patch release because there will be another patch release soon. 

So I'll reach out for Ryan for an opinion. If we don't get back to you in time, just upgrade and we'll align the versions later. 

@ryanemerson - can you please let me know on which issue in ISPN we are waiting? Is there anything that would keep us from upgrading except of the JGroup metrics, which sound harmless to me at least in the KCB benachmark project as we don't rely much on those metrics at the moment? We might need to update a dashboard or two though when we now do your load and performance tests.

Reply:
tldr; It's should be safe to merge to Infinispan 15.0.6.Final, but 15.0.7.Final will be available shortly.

I think there's a little confusion here, I'll try to explain. The metrics incompatibility issue was at the JGroups level [JGP-2814](https://issues.redhat.com/browse/JGRP-2814), which is what has been delaying Infinispan 15.0.7.Final. We couldn't avoid a JGroups upgrade, as we needed [JGRP-2813](https://issues.redhat.com/browse/JGRP-2813)  to resolve keycloak/keycloak-benchmark#889.

Whilst a JGroups metric incompatibility is not an issue for the KCB project, it could cause issues for other Infinispan users and like Keycloak we're trying to improve user's upgrade experience, hence why we had to delay.

JGroups issue should now be resolved and I hope to release Infinispan 15.0.7.Final later today.

Reply:
@ahus1 Thanks for the feedback. I reverted the ISPN upgrade for now.

Reply:
@ahus1 Seems an [ISPN test is failing](https://github.com/keycloak/keycloak/actions/runs/10265585450/job/28405547785?pr=31678) without the upgrade. We might need to do it in the end...

Reply:
The test ExternalInfinispanTest.testLoadBalancerCheckFailure is unfortunately unstable. I've created issue https://github.com/keycloak/keycloak/issues/31972

It is only unstable in main. As the PR will test this with a provisional rebase AFAIK, you will only see this once you rebase this PR.

Reply:
@vmuzikar - the Infinispan upgrade has landed in main, together with other changes to this test. 
Please rebase the PR.